cypress/page/shoping-cart.page.ts

```diff
@@ -1,14 +1,10 @@
 class ShopingCartPage{
-    private buttonContainer : string;
+
     private goTologin : string;
 
     constructor(){
-        this.buttonContainer = "[style*=' display: block;'] .button-container > a";
-        this.goTologin = ".cart_navigation span";
-    }
 
-    public clickButtonContainer():void{
-        cy.get(this.buttonContainer).click();
+        this.goTologin = ".cart_navigation span";
```

Comment:
here instead of the span, you can use an attribute to indicate we are looking for the proceed to checkout button.